README.md

```diff
@@ -1,4 +1,88 @@
+  <h3><b>Hello Microverse Project</b></h3>
 
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
+**[Hello Microverse]** is the first project in the first block of the first module of Microverse full stack web development program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Kindly make sure the Getting Started section has all sub sections set according to the [Microverse Readme Template](https://github.com/microverseinc/readme-template). I would recommend you to stick to the review your last reviewer gave you [here](https://github.com/H-Ramalan/Hello-Microverse/pull/2#discussion_r1178853517)